laser_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: laser_joint
    joint_speed: 1.17
    motor:
        id: 1
        init: 0
        min: 0
        max: 3000
